apps/pie-docs/src/foundations/spacing/overview.md

```diff
@@ -96,20 +96,20 @@ Here are some examples of use which might help guide your decisions when applyin
 
 ### Layout spacing
 
-Horizontal spacing within design layouts should always try to adhere to our predefined Grids.  As we follow an element first approach, try to expand your components to fill up the necessary columns within the Grid.
+Horizontal spacing within design layouts should always try to adhere to our predefined grids.  As we follow an element-first approach, try to expand your components to fill up the necessary columns within the Grid.
 
-Regarding vertical spacing, make sure to use spacing as a way to create meaningful blocks of content, using bigger and smaller spacing measures to improve the visual hierarchy of your layout and to keep a consisten vertical rythm within the page.
+Regarding vertical spacing, make sure to use spacing as a way to create meaningful blocks of content, using bigger and smaller spacing measures to improve the visual hierarchy of your layout and to keep a consistent vertical rhythm within the page.
```

Comment:
```suggestion
Make sure to use vertical spacing as a way to create meaningful blocks of content. Using bigger and smaller spaces to improve the visual hierarchy of your layout and to keep a consistent vertical rhythm within the page.
```

Reply:
👍

Reply:
@FayeCarter I updated this one again, the original was more correct than my first suggestion.

Reply:
👍